dangerzone/gui/main_window.py

```diff
@@ -735,6 +746,34 @@ def handle_wsl_install_failed(self, msg: str) -> None:
         self.status_bar.handle_startup_error()
         self.show_wsl_error_widget()
 
+    def handle_needs_user_input_install_remote_container(
+        self, req: startup.PromptRequest
+    ) -> None:
+        log.debug("Prompting user to download new remote container")
+        question = Question(
+            self.dangerzone,
+            title="Download Sandbox",
+            message=DOWNLOAD_NEW_CONTAINER_IMAGE_MSG,
+            ok_text="OK",
+            cancel_text="Skip for now (don't)",
+            checkbox_text="Always download Sandbox updates",
+        )
+        assert question.checkbox is not None
+        result = question.launch()
+        if question.checkbox.isChecked():
+            self.dangerzone.settings.set(
+                "updater_ask_before_download",
+                False,
+                autosave=True,
+            )
+        if result == Question.Accepted:
+            log.debug("Accepted")
+            req.req_data["signal"].emit()
```

Comment:
Do we need to manually emit a signal, if the `_download_container` method will do it for us?